docker-compose.yaml

```diff
@@ -34,10 +34,46 @@ services:
       - 4224:4222
       - 8224:8222
     volumes:
-      - ./nats3.conf:/etc/nats/nats-server.conf
+      - ./configs/nats3.conf:/etc/nats/nats-server.conf
       - ./shared.conf:/etc/nats/shared.conf
       - nats3-data:/data
 
+  nex1:
+    image: registry.synadia.io/nexce:main
```

Comment:
using `main` tag temporarily until I can cut a new release, which should be this week.  After that, `latest` will always be good enough